### ISSUE TITLE
missing scene.obj

### ISSUE THREAD

**Issue by oscarbg** [GPUOpen-LibrariesAndSDKs/RadeonRays_SDK#208]
Hi,
compiling tutorial_vk needs scene.obj file.. dx same..
can include simple obj..
thanks..

**Comment by himanshugoel2797** [comment]
I just used the sponza.obj from here https://github.com/jimmiebergmann/Sponza

**Comment by igpooh** [comment]
We probably add a simple scene a bit later. Sponza is a good one for experiments as mentioned above.

**Comment by oscarbg** [comment]
Ok, thanks for help..
@igpooh I don't want to open another issue but once you select a default ".obj" for the samples to run would be nice to add to README.md some expected performance numbers on Mrays/s on various AMD hardware.. for example on a Vega 56 I was getting from a simple "cube.obj" ( seeing the console output) something like 450Mrays/s and on Titan V >1000Mrays/s if I remember correctly..
other issue I could open (altough lame one) is that one sample backend seems missing printing to console the Mrays/s.. I believe VK prints Mrays/s but DX12 backend not..
I know RadeonRays is for devs but anyway for "curious" users would be nice to have this performance numbers on various GPUs and compare performance differences of DX12 and Vulkan backends.. 
sorry for long response..
will be closing later once scene is added if you don't mind..

thanks..

**Comment by norsetto** [comment]
Still have to check the code, but is this what one should get with the sponza scene?
![vk_geom](https://user-images.githubusercontent.com/3383359/82121000-2fb69f80-978a-11ea-9a63-c7f3cab39e12.jpg)

**Comment by himanshugoel2797** [comment]
Yep. That's exactly what I get. The colors are the UV along the triangle.

**Comment by igpooh** [comment]
> Ok, thanks for help..
> @igpooh I don't want to open another issue but once you select a default ".obj" for the samples to run would be nice to add to README.md some expected performance numbers on Mrays/s on various AMD hardware.. for example on a Vega 56 I was getting from a simple "cube.obj" ( seeing the console output) something like 450Mrays/s and on Titan V >1000Mrays/s if I remember correctly..
> other issue I could open (altough lame one) is that one sample backend seems missing printing to console the Mrays/s.. I believe VK prints Mrays/s but DX12 backend not..
> I know RadeonRays is for devs but anyway for "curious" users would be nice to have this performance numbers on various GPUs and compare performance differences of DX12 and Vulkan backends..
> sorry for long response..
> will be closing later once scene is added if you don't mind..
> 
> thanks..

Yes, sure, I will add the performance results printing for DX12 in the upcoming version. Some performance numbers will be included too.

**Comment by oscarbg** [comment]
@igpooh thanks..
